scripts/deploy_assisted_service.sh

```diff
@@ -67,8 +68,10 @@ if [ "${OPENSHIFT_INSTALL_RELEASE_IMAGE}" != "" ]; then
     fi
 fi
 
-OS_IMAGES=$(skipper run ./scripts/override_os_images.py --src ./assisted-service/data/default_os_images.json)
-export OS_IMAGES
+if [ "${OPENSHIFT_CI}" == "true" ]; then
```

Comment:
This line compares variable is  string `true`  which fail if we set it to `True`

it's better this way this wil ensure that `true` or `True` even worse `TRUE`  will be correct 
```suggestion
if [ [ "${OPENSHIFT_CI}" ] ]; then
```

Reply:
This variable is set by the CI, so I guess there is no risk here. Also https://github.com/openshift/assisted-test-infra/blob/1209bd8ce092e3043e3b89fe06e1e58c0aa942ac/scripts/bring_assisted_service.sh#L42

Reply:
yes I dont like it either -- so for the CI is none issue -- but it still best practice 
again becuse we have it all over the place it OK. just saying it better to use it as `bool` check and not `str`